ls1intum/Artemis
# `Assessment`: Add general feedback in example submissions

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

[ ] * - not needed

### Motivation and Context
When tutors are assessing students' submission, they are able to create general (unreferenced) feedback.
However, this was not possible when tutors were assessing example submissions.
In this PR, we add the possibility to create general feedback for both text and modelling example submissions.

### Description
Added unreferenced-feedback to both text and modeling example submission editors.
Adjusted the tutor training logic to work with unreferenced feedback. 

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Tutor

1. Log in to Artemis as **Instructor**
2. Navigate to Course Management
3. Navigate to a text or modeling exercise
4. Create an example submission and example assessment with general feedback. Check **Tutors need to assess this submission** checkbox

1. Log in as **Tutor**
2. Navigate to Exercise Dashboard of the above mentioned exercise
3. Assess the example submission and observe that the system's messages are displayed as expected (tutor training logic is correct, see screenshot)

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


Tutor assessing an example submission and receiving systems' status for each feedback.
<img width="1858" alt="after" src="https://user-images.githubusercontent.com/51077603/143968790-27f28431-82a0-4958-98c8-43a688bbeab0.png">
